lib/l10n/intl_es.arb

```diff
@@ -0,0 +1,438 @@
+{
```

Comment:
This is a copy of intl_messages, which was automatically generated, where I've translated a few strings to test.